src/ui/screens/ScreenGetHelp.qml

```diff
@@ -16,13 +17,6 @@ Item {
         color: window.color
     }
 
-    VPNMenu {
-        id: menu
-        objectName: "getHelpBack"
-        _menuOnBackClicked: () => VPNNavigator.requestPreviousScreen()
-        _menuIconButtonSource: getHelpStackView.depth === 1 ? "qrc:/nebula/resources/close-dark.svg" : "qrc:/nebula/resources/back.svg"
-    }
-
     VPNMouseArea {
```

Comment:
I think we'll need to rehome this `MouseArea`, which seems to have been orphaned by the move of `VPNMenu`. I can't trigger the dev menu and seeing this in console: `Warning: qrc:/ui/screens/ScreenGetHelp.qml:20:5: QML VPNMouseArea: Cannot anchor to an item that isn't a parent or sibling. (ScreenGetHelp.qml:20)`